sdk/README.md

```diff
@@ -1,12 +1,12 @@
 # Pulumi Language SDKs
 
-This directory contains the Pulumi SDKs for all supported languages.
-
-Please see the respective `README`s for information about installing and using these libraries:
+This directory contains Pulumi SDKs for the following languages:
 
 * [Node.js](./nodejs)
 * [Python](./python)
 * [Go](./go)
 * [.NET](./dotnet)
 
+Please see the respective `README`s for information about installing and using these libraries.
```

Comment:
```suggestion
See the respective `README`s for information about installing and using these libraries.
```

there's deets in the style guide in hugo, but we should try and avoid the use of `please`